tests/acceptance/21_methods/call_methods_using_array_expansion.cf

```diff
@@ -0,0 +1,66 @@
+#######################################################
+#
+# Redmine 6369 - make sure that arrays don't accumulate
+#
```

Comment:
Redmine??? I know that's an old bug system but why mentioned here? Shouldn't it be CFE-636 mentioned here?

Reply:
@craigcomstock @nickanderson Feel free to fix this in a follow-up.